resources/build.gradle.kts

```diff
@@ -15,6 +16,14 @@ group = "dev.icerock.moko"
 version = libs.versions.mokoResourcesVersion.get()
 
 kotlin {
+    sourceSets {
+        val commonMain by getting
+        val jvmMain by getting {
+            dependencies {
+                implementation(libs.icu4j)
```

Comment:
dependency should be declared in dependencies block below

Reply:
fixed